plugins/command-not-found/command-not-found.plugin.zsh

```diff
@@ -23,3 +23,13 @@ if [ -f /usr/libexec/pk-command-not-found ]; then
         return $retval
     }
 fi
+
+# OSX command-not-found support
+# https://github.com/Homebrew/homebrew-command-not-found
+if which brew > /dev/null; then
+  if brew tap | command grep -q homebrew/command-not-found; then
+    if brew command command-not-found-init > /dev/null; then
```

Comment:
Not a problem, I was a little bit confused but xu-cheng explained further to clarify.  Thanks!